exonum/src/blockchain/block.rs

```diff
@@ -46,12 +47,18 @@ encoding_struct!(
     }
 );
 
-/// Block with pre-commits.
+/// Block with its Precommit messages.
```

Comment:
Nitpick: It may make sense to format "Precommit" as code with the help of ticks `.